prep-yml-bundle/rename-kube-domain.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+function usage() {
+  echo "Usage: ./rename-kube-domain.sh DOMAIN path/to/yamls path/to/scf path/to/uaa-fissile-release"
```

Comment:
You're going to need to expand on this to say what those arguments mean…

Reply:
:man_facepalming: from here down can be ignored. These files should have been deleted now that we're using helm.